### PR TITLE
Fixes sort algorithm so breadcrumb truncation will work in IE

### PR DESCRIPTION
**Pull request by djsauble**
Issue: https://issues.sonatype.org/browse/NEXUS-8203

Sort function expects return value of 1, 0, and -1 when `a` is less than, equal to, or greater than `b`, respectively. I had been returning true/false instead, which somehow worked in Chrome/Firefox/Safari, but broke IE.

### CODE REVIEW

File: plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/drilldown/Drilldown.js [hunk 1]
```diff
@@ -492,17 +492,24 @@ Ext.define('NX.view.drilldown.Drilldown', {
...
    */
   reduceButtonWidth: function(buttons, width, minPerButton) {
     var me = this,
       currentWidth = 0,
-      currentButtonWidth,
-      nextButtonWidth,
       setToWidth;
 
+    // Sort the buttons by width
+    buttons = buttons.sort(
```
Comment (kellyrob99) [inline plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/drilldown/Drilldown.js]: should be able to simplify to:
```return a.getWidth() - b.getWidth()```

**Comment by kellyrob99** [comment]
+1

File: plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/drilldown/Drilldown.js (hunk 1 above)
Comment (djsauble) [inline plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/drilldown/Drilldown.js]: I’m a sucker for simplification. Amended.

**Comment by djsauble** [comment]
Manual merge.

**Pull request closed without merging**